tests/bsim/bluetooth/tester/tests_scripts/gap.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Copyright 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Smoketest for GAP BTP commands with the BT tester
+
+simulation_id="tester_gap"
+verbosity_level=2
+EXECUTE_TIMEOUT=20
```

Comment:
```suggestion
EXECUTE_TIMEOUT=100
```
It fails because it is timing out :)